data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/command/dataexport/ExportCommand.java

```diff
@@ -1,17 +1,66 @@
 package com.scalar.db.dataloader.cli.command.dataexport;
 
+import com.scalar.db.dataloader.cli.exception.DirectoryValidationException;
+import com.scalar.db.dataloader.cli.exception.InvalidFileExtensionException;
+import com.scalar.db.dataloader.cli.util.DirectoryUtils;
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
 import java.util.concurrent.Callable;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang3.StringUtils;
 import picocli.CommandLine;
 import picocli.CommandLine.Model.CommandSpec;
 import picocli.CommandLine.Spec;
 
 @CommandLine.Command(name = "export", description = "Export data from a ScalarDB table")
 public class ExportCommand extends ExportCommandOptions implements Callable<Integer> {
 
+  private static final List<String> ALLOWED_EXTENSIONS = Arrays.asList("csv", "json", "jsonl");
+
   @Spec CommandSpec spec;
 
   @Override
   public Integer call() throws Exception {
+    validateOutputDirectory(outputFilePath);
     return 0;
   }
+
+  private void validateOutputDirectory(String path)
```

Comment:
[minor]

```suggestion
  private void validateOutputDirectory(@Nullable String path)
```

`import javax.annotation.Nullable` is also needed.

Reply:
Updated and added in the code.